docs/use-cases/tokenization/index.page.tsx

```diff
@@ -0,0 +1,395 @@
+    </div>
+  );
+};
+
+export default function Tokenization() {
+  const modalRef = useRef(null);
+
+  return (
+    <div className="landing page-tokenization">
+      <div className="position-relative"></div>
+      <div className="position-relative d-none-sm"></div>
+      <section className="container-new mt-16">
+        <div className="card-grid card-grid-2xN">
+          <div className="col">
+            <img className="tokenization-graphic mw-100 hide-md" />
+          </div>
+          <div className="col">
+            <div className="d-flex flex-column-reverse">
+              <img className="tokenization-graphic mw-100 show-md" />
+              <h2 className="h4 h2-sm mb-10">
+                {translate(
+                  "Work with a variety of tokens supported by the XRP Ledger."
+                )}
+              </h2>
+              <h6 className="eyebrow mb-3 text-large">
+                {translate("Tokenization")}
+              </h6>
+            </div>
+            <p className="mb-10">
+              {translate(
+                "Explore the possibilities of tokenization on the XRP Ledger, revolutionizing ownership, transactions, and value exchange with unparalleled efficiency and security."
+              )}
+            </p>
+            <div className="tokenization-color-bar" />
+            <div className="mb-10">
+              <NavList pages={useCases} bottomBorder={false} />
+            </div>
+            <div className="d-flex">
+              <a
+                className="btn btn-primary d-inline-flex align-items-center"
+                href="/docs"
+              >
+                {translate("Quick Start")}
+              </a>{" "}
+              <a
+                className="ml-4 video-external-link btn-none"
+                target="_blank"
+                href="https://www.youtube.com/playlist?list=PLJQ55Tj1hIVZtJ_JdTvSum2qMTsedWkNi"
+              >
+                <i className="fa fa-video-camera" />
+                <span className="link-text">{translate("Learn more")}</span>
+              </a>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section className="container-new py-20">
+        <div className="d-flex flex-column-reverse">
+          <p className="mb-16">
+            {translate(
+              "Check out the security features you can tap into right from the chain without the hassle of piecing together smart contracts."
+            )}
+          </p>
+          <h4 className="eyebrow mb-8">{translate("Security advantages")}</h4>
+        </div>
+        <div
+          className="security-card-grid nav card-grid"
+          id="security-features"
+        >
+          {securityAdvantages.map((advantage) => (
+            <div className="security-card" key={advantage.id}>
+              <div className="card-body p-6">
+                <h4 className="card-title h6">{translate(advantage.title)}</h4>
+                <p>{advantage.description}</p>
```

Comment:
This produces invalid markup because the description sections often contain `<div>` tags which are not allowed to nest inside `<p>` tags. Suggested fix:

```suggestion
                <div>{advantage.description}</div>
```